src/main/scala/Main.scala

```diff
@@ -1,33 +1,34 @@
-def main(args: Array[String]): Unit = {
+object Main:
```

Comment:
I missed this one in my previous review. We can update it to use the simplest for of `main` definition
```diff
- def main(args: Array[String]): Unit = {
+ @main def Main: Unit = {
```

Reply:
`@main`  didnt work for some reason :(